base/feature_override_unittest.cc

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/feature_override.h"
+#include "base/feature_list.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace base {
+namespace {
+
+const Feature kTestControlEnabledFeature{"TestControlEnabledFeature",
+                                         FEATURE_ENABLED_BY_DEFAULT};
+const Feature kTestControlDisabledFeature{"TestControlDisabledFeature",
+                                          FEATURE_DISABLED_BY_DEFAULT};
+
+const Feature kTestEnabledButOverridenFeature{"TestEnabledButOverridenFeature",
+                                              FEATURE_DISABLED_BY_DEFAULT};
+const Feature kTestDisabledButOverridenFeature{
+    "TestDisabledButOverridenFeature", FEATURE_ENABLED_BY_DEFAULT};
+
+constexpr Feature kTestConstexprEnabledButOverridenFeature{
+    "TestConstexprEnabledButOverridenFeature", FEATURE_DISABLED_BY_DEFAULT};
+constexpr Feature kTestConstexprDisabledButOverridenFeature{
+    "TestConstexprDisabledButOverridenFeature", FEATURE_ENABLED_BY_DEFAULT};
+
+ENABLE_FEATURE_BY_DEFAULT(kTestDisabledButOverridenFeature);
+ENABLE_FEATURE_BY_DEFAULT(kTestConstexprDisabledButOverridenFeature);
+DISABLE_FEATURE_BY_DEFAULT(kTestEnabledButOverridenFeature);
```

Comment:
Where are these `ENABLE_FEATURE_BY_DEFAULT` and `DISABLE_FEATURE_BY_DEFAULT` macros defined?